File: 백준/Bronze/2675. 문자열 반복/문자열 반복.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

let initialInput = true;
let number;
let count = 0;
readline
  .on("line", function (line) {
    if (initialInput) {
      number = Number(line);
      initialInput = false;
    } else {
      const [repeatNumber, str] = line.split(" ");
      console.log([...str].map((str) => str.repeat(repeatNumber)).join(""));

      count++;
      if (number === count) readline.close();
    }
  })
  .on("close", function () {
    process.exit();
  });

File: 백준/Bronze/2675. 문자열 반복/문자열 반복.py
# 문자열 반복 2675 브론즈 2

def solution_1():
    n = int(input())
    for _ in range(n):
        k, string = input().split(" ")
        k = int(k)
        answer = ''
        for i in string:
            answer += i * k
        print(answer)
solution_1()
